mvc/application/models/MakePost.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class MakePost extends ConnectDB {
+  public function addPost($postContent)
+  {
+    $t = time();
+    $postId = date("Y-m-d H:i:s", $t) . " " . $_SESSION["userId"];
+    $postAuthorId = $_SESSION["userId"];
+    $postAuthor = $_SESSION["userName"];
+    $sql = "insert into post (postId,postAuthor,postContent,postAuthorId) values('$postId','$postAuthor','$postContent','$postAuthorId');";
+    $this->query($sql);
+    return true;
+  }
+}
```

Comment:
If leaving a blank line at the beginning of the code make sure to maintain the same at the end of the code.